specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -572,8 +578,11 @@
               "type": "string",
               "description": "It provides the Timestamp for when the health impacting event was last updated.",
               "format": "date-time"
+            },
+            "hIRStage": {
```

Comment:
hIR -> hir? not sue if the whole acronym is supposed to be lower case when it leads

Reply:
Is there a way I can confirm what it should be?